src/contexts/documentationEditorContext.js

```diff
@@ -0,0 +1,47 @@
+import { createDocumentationRequest } from '@/services/documentationEditorService';
+import { useRouter } from 'next/router';
+import { createContext, useContext, useState } from 'react';
+
+export const DocumentationEditorContext = createContext({});
+
+export function DocumentationEditorProvider({ children }) {
+  const router = useRouter();
+  const [isLoading, setIsLoading] = useState(false);
+
+  async function createDocumentation({ title, content, token }) {
+    setIsLoading(true);
+
+    const { status } = await createDocumentationRequest({
+      title,
+      content,
+      token,
+    });
+
+    if (!status) {
+      setIsLoading(false);
+      return {
+        message:

```

Comment:
suggestion: could not be errorMessage to be more clear that is a error message and not a generic message?